### PR TITLE
Add notes about the new way to deal with primary key introduced in the migrations plugin

### PR DESCRIPTION
**Pull request by HavokInspiration**
To be merged when the PR https://github.com/cakephp/migrations/pull/116 is merged

### CODE REVIEW

**Comment by lorenzo** [comment]
Could you add this to the book as well when the other PR gets merged?

**Comment by HavokInspiration** [comment]
@lorenzo What do you mean ? I'm in the book already :smile:

**Comment by HavokInspiration** [comment]
@lorenzo What do you mean ? I'm in the book already :smile: I can add it to the README of the plugin however

File: fr/migrations.rst [hunk 1]
```diff
@@ -247,6 +247,48 @@ méthode ``table()``::
...
+Comparée à la méthode précédente de gestion des clés primaires, cette méthode
+vous donne un plus grand contrôle sur la définition de la colonne de la clé
+primaire : signée ou non, limite, commentaire, etc.
+
+Toutes les migrations et les snapshots créés avec ``bake`` utiliseront cette
+nouvelle méthode si nécessaire.
+
+.. warning::
+
+    Gérer les clés primaires ne peut être fait que sur pour créations de
```
Comment (antograssiot) [inline fr/migrations.rst]: que sur pour ? :wine_glass: :wink:

Comment (HavokInspiration) [inline fr/migrations.rst]: Mieux... "que sur pour créations de tables".

**Comment by lorenzo** [comment]
lol, my bad

**Comment by antograssiot** [comment]
merging as cakephp/migrations#116 has been merged now

**Pull request merged**